src/cmd/tools/m3db_tool/README.md

```diff
@@ -0,0 +1,62 @@
+M3DB Tool
+========
+
+This is a CLI tool to do some things that may be desirable for cluster introspection, or for operational purposes.
+
+Where configuration data is required its provided via YAML.
+
+You can:
+
+* create a database per the simplified database create API
+* list namespaces
+* delete namespaces
+* list placements
+* delete placements
+* add nodes
+* remove nodes
+
+NOTE: This tool can delete namespaces and placements.  It can be quite hazardous if used without adequate understanding of your m3db cluster's topology, or without a decent understanding of how m3db works.
+
+Examples
+-------
+
+    # show help
+    m3db-tool -h
```

Comment:
I'd suggest a different name. Maybe `m3ctl` 💭

Reply:
i like that better too